.zshrc

```diff
@@ -18,7 +17,6 @@ plugins=(git)
 source $ZSH/oh-my-zsh.sh
 
 # git autocompletion
-$HOME/.zsh/gitprompt.sh
```

Comment:
This appeared to be something within the `~/.zsh/` folder on my old work machine that isn't available immediately on initializing new machine, removing until deemed necessary.

Note: may be related to installation of extra `git-prompt` dependency
- _[bash](https://github.com/magicmonty/bash-git-prompt)_
- _[zsh](https://github.com/olivierverdier/zsh-git-prompt)_